carts/src/test/java/com/oracle/coherence/examples/sockshop/micronaut/carts/TracingTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at
+ * https://oss.oracle.com/licenses/upl.
+ */
+
+package com.oracle.coherence.examples.sockshop.micronaut.carts;
+
+import com.oracle.coherence.common.base.Blocking;
+import com.tangosol.net.CacheFactory;
+import com.tangosol.util.Base;
+import io.jaegertracing.internal.JaegerSpan;
+import io.micronaut.context.annotation.Property;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.runtime.server.EmbeddedServer;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.hamcrest.core.Is;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@MicronautTest
+@Property(name = "tracing.jaeger.enabled", value = "true")
+public class TracingTest {
+
+	@Inject
+	EmbeddedServer server;
+
+	@Inject
+	TracerFactory.SafeInMemoryReporter reporter;
+
+	@BeforeEach
+	void setup() {
+		RestAssured.reset();
```

Comment:
Even though there is only one test, I would recommend clearing the `SafeInMemoryReporter` before each test is run.

Reply:
Fixed.